tests/conftest.py

```diff
@@ -0,0 +1,67 @@
+"""Copyright (c) 2017, Anthony Scopatz
+All rights reserved."""
+import os
+import shutil
+import subprocess
+import sys
+import tempfile
+
+import pytest
+import json
```

Comment:
json is part of the standard library and should be grouped with the first set of imports